src/model/cloud-runner/services/core/task-parameter-serializer.ts

```
@@ -91,6 +93,28 @@ export class TaskParameterSerializer {
     return TaskParameterSerializer.serializeFromType(CloudRunnerOptions);
   }
 
+  private static serializeAwsEnvironmentVariables() {
+    const awsEnvVars = [
```

ESLint

[error] The variable `awsEnvVars` should be named `awsEnvironmentVariables`. A more descriptive name will do too.

(unicorn/prevent-abbreviations)